docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
```

Comment:
PR https://github.com/telefonicaid/fiware-orion/pull/1059 contains a very good .md document that could be used as template for the Ubuntu case.